docs/visualization/visualize_2p_responses.ipynb

```diff
@@ -38,7 +38,7 @@
   },
```

Comment:
<p>Before pushing make sure to change your api key back to the <code>os.environ</code> line </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>